OpenRA.Mods.Common/Widgets/SpritePowerMeterWidget.cs

```diff
@@ -0,0 +1,119 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2020 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System;
+using OpenRA.Graphics;
+using OpenRA.Widgets;
+
+namespace OpenRA.Mods.Common.Widgets
+{
+	public class SpritePowerMeterWidget : Widget
+	{
+		[Desc("Height of each meter bar")]
+		[FieldLoader.Require]
+		public readonly int MeterHeight = 4;
```

Comment:
The other names/comments show that "Meter" refers to the collection as a whole and "bar" to the individual blips. This also appears to be used as the stride between bar segments rather than the height of a specific bar element.

This should therefore probably be called `BarStride`?